shipwright/multi-arch-native-build-strategy.yaml

```diff
@@ -0,0 +1,539 @@
+---
```

Comment:
This is a standard task, no OKD-specific modifications required?

Reply:
Yes, it's an object to apply in a generic kubernetes cluster where Shipwright and Tekton are installed.